src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -179,4 +182,38 @@ describe('NameAndPictureScreen', () => {
     expect(getByText('nameAndPicGuideCopyTitle')).toBeTruthy()
     expect(getByText('nameAndPicGuideCopyContent')).toBeTruthy()
   })
+  it('does not render skip button when configured so', () => {
+    // TODO add mock Statsig flag
+    const { queryByText } = render(
+      <Provider store={createMockStore()}>
+        <MockedNavigator component={NameAndPicture} />
+      </Provider>
+    )
+    expect(queryByText('skip')).toBeNull()
+  })
+  // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
the linter encodes a Valora style guide, and shouldn't be disabled unless there's a really good reason for it. In this case, `it.todo` could be used instead, so I don't see a reason to disable linting and use `it.skip`